bgpd/bgp_packet.c

```diff
@@ -2814,6 +2814,8 @@ static int bgp_capability_msg_parse(struct peer *peer, uint8_t *pnt,
 		if (CHECK_FLAG(peer->flags, PEER_FLAG_OVERRIDE_CAPABILITY))
 			continue;
 
+		capability = lookup_msg(capcode_str, hdr->code, NULL);
```

Comment:
hmm - should the 'default' be an empty string instead of a NULL - would that be safer?

Reply:
Makes sense, will update, thanks.

Reply:
Updated.